packages/client/test/sync/fullsync.spec.ts

```diff
@@ -26,7 +26,6 @@ tape('[FullSynchronizer]', async (t) => {
     const pool = new PeerPool() as any
     const chain = new Chain({ config })
     const sync = new FullSynchronizer({ config, pool, chain })
-    pool.emit('added', { eth: true })
```

Comment:
Have you triple-checked that this is not used anywhere?

Reply:
yes this was removed with the central event bus refactoring

Reply:
👍